package.json

```diff
@@ -9,7 +9,8 @@
   "types": "dist/index.d.ts",
   "scripts": {
     "clean": "rm -rf dist",
-    "lint": "eslint . --ext .ts,.js && tsc --noEmit",
+    "example": "npm run build && cp ./dist/bundle.js ./examples/dist/bundle.js && http-server -c-1 ./examples",
```

Comment:
I went for no-config on the example server, so it wasn't able to find the `../dist/bundle.js` because http-server only serves the folder it's pointed to.

However, it's my mistake I didn't commit the `mkdir` and it makes sense to use a preexample here for sure 👍